pkg/tests/mappings/all_test.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
```

Comment:
This file might be better called `manifest_all_test.py`.  Might also be worth putting the manifest tests in their own package, especially if this is to become the common packaging backend in the future.

Reply:
That is a stylistic mistake in Starlark that I am just never going to get used to.
Done

Reply:
Could you elaborate on this? The updates look good, just curious on my end.